See blog/see-blog.css

```diff
@@ -1,6 +1,210 @@
+  width: 24px;
+  height: 24px;
+  left: 359px;
+  top: 50px;
+  margin-top: 4px;
+  padding-left: 10px;
+}
+.top-par {
+  font-family: Metrophobic, bold;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 150%;
+  width: 305px;
+  height: 50px;
+  left: 16px;
+  top: 94px;
+  margin-left: 16px;
+  margin-top: 20px;
+}
+.person {
+  margin-left: 16px;
+  width: 50px;
+  height: 50px;
+}
+.author-info {
+  width: 142px;
+  height: 32px;
+  left: 82px;
+  top: 173px;
+}
+.author {
+  display: flex;
+  margin-top: 13px;
+}
+.time {
+  font-size: 12px;
+  opacity: 0.7;
+}
+
+.photo {
+  margin-left: 16px;
+  width: 343px;
+  height: 178px;
+  left: 16px;
+  top: 234px;
+  background: url(../images/Rectangle\ 1485.png);
+  border-radius: 10px;
+}
+.blog {
+  width: 343px;
+  height: 344px;
+  font-family: Metropolis;
+  font-size: 13px;
+  line-height: 150%;
+  margin-left: 16px;
+  margin-top: 15px;
+  color: #696969;
+}
+
+@media (min-width: 1200px) {
+  .left-side {
+    margin-left: 150px;
+  }
+  .right-side {
+    margin-right: 150px;
+  }
+  .top-par {
+    margin-left: 150px;
+    font-family: Metrophobic, bold;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 35px;
+    line-height: 150%;
+    display: inline;
+    text-align: center;
+  }
+  .author {
+    margin-left: 180px;
+    margin-top: 20px;
+    width: 3000px;
+    height: 100px;
+  }
+  .photo {
+    margin-left: 150px;
+    width: 1000px;
+    height: 500px;
+    background: url(../images/Rectangle\ 1485.png);
+    margin-top: -0.5px;
+    background-size: cover;
+  }
+  .blog {
+    margin-left: 150px;
+    width: 1000px;
+    height: 300px;
+    font-size: 25px;
+    line-height: 150%;
+  }
+}
+@media (min-width: 992px) and (max-width: 1200px) {
+  .left-side {
+    margin-left: 150px;
+  }
+  .right-side {
+    margin-right: 100px;
+  }
+  .top-par {
+    margin-left: 150px;
+    font-family: Metrophobic, bold;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 28px;
+    line-height: 150%;
+    display: inline;
+    text-align: center;
+  }
+  .author {
+    margin-left: 122px;
+    margin-top: 30px;
+  }
+  .photo {
+    margin-left: 122px;
+    width: 800px;
+    height: 500px;
+    background: url(../images/Rectangle\ 1485.png);
+    margin-top: 20px;
+    background-size: cover;
+  }
+  .blog {
+    margin-left: 122px;
+    width: 800px;
+    height: 200px;
+    font-size: 25px;
+    line-height: 150%;
+  }
+}
+@media (min-width: 600px) and (max-width: 992px) {
+  .left-side {
+    margin-left: 120px;
+  }
+  .right-side {
+    margin-right: 90px;
+  }
+  .top-par {
+    margin-left: 120px;
+    font-family: Metrophobic, bold;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 150%;
+    display: inline;
+    text-align: center;
+  }
+  .author {
+    margin-left: 100px;
+    margin-top: 30px;
+  }
+  .photo {
+    margin-left: 100px;
+    width: 600px;
+    height: 500px;
```

Comment:
You need to fix the size of this photo in tablet size, I think `height: 300px` will be suitable